src/components/Avatar/Avatar.vue

```diff
@@ -54,7 +54,10 @@
 		<div v-if="hasMenu" class="icon-more" />
 
 		<!-- avatar status -->
-		<div v-if="status"
+		<div v-if="canDisplayUserStatus"
+			class="avatardiv__user-status"
+			:class="'avatardiv__user-status--' + userStatus.status" />
+		<div v-else-if="status"
 			class="avatardiv__status"
 			:class="'avatardiv__status--' + status"
 			:style="{ backgroundColor: `#${statusColor}` }">
```

Comment:
Is it still worth having the old status?
I'd rather have this as the new design you implemented :)

Reply:
Right now they exist along, but the user-status takes priority.
We should mark it as deprecated and remove it with nc/vue 3.0 i would say.

Does that sound alright?

Reply:
Sure thing! :)

Reply:
Done